services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from typing import Optional
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: Optional[str] = None) -> list:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> Optional[MovieSession]:
+    return MovieSession.objects.get(id=movie_session_id)
```

Comment:
The `get_movie_session_by_id` function uses `MovieSession.objects.get`, which will raise a `DoesNotExist` exception if no matching record is found. Consider using `MovieSession.objects.filter(id=movie_session_id).first()` to return `None` if the session does not exist, or handle the exception appropriately.